### ISSUE TITLE
Why microphone permission was asked

### ISSUE THREAD

**Issue by Jean-BaptisteC** [nobelization/panoramax-mobile-app#68]
When the user wants to take a sequence for the first time,  the app asks to get access to the microphone.
IMO this permission is not necessary to take a sequence.
Keep in mind, that when you publish an app on the store, Google asks you to tell why you need this permission (Users can also ask the same question).
I have seen this permission was not declared in the Android manifest, this permission is asked by the camera library?

**Comment by OtterWays** [comment]
Yes, this permission is linked to the camera permission. In fact, this won't prevent the publication on the Play Store since the mechanism is automatic. However, it is possible to disable the permission request to avoid disturbing users. Note that the permission will still be visible in the app's information, but its default value will be "denied".

However, the permission is not requested on iOS.

**Comment by Jean-BaptisteC** [comment]
Thanks for detail
Maybe we can hide permission in manifest with `tools:node="remove"` like this?

https://github.com/Exodus-Privacy/exodus-android-app/blob/c38f092316105ccc520a199bf8356d6ccebd5fbb/app/src/main/AndroidManifest.xml#L9

**Comment by chrisdebian** [comment]
Hi, all. Has this been merged now; can we close this ticket.

Chris
